mesonbuild/modules/windows.py

```diff
@@ -30,6 +30,7 @@ def detect_compiler(self, compilers):
                 return compilers[l]
         raise MesonException('Resource compilation requires a C or C++ compiler.')
 
+    @FeatureNewKwargs('windows.compile_resources', '0.47.0', ['depend_files'])
```

Comment:
Thanks, I was just looking a how to do this.

I don't think this works, because compile_resources() doesn't take any kwargs, it has a positional parameter called 'kwargs' which is expected to be a dict, but the FeatureNewKwargs decorator is written to look at the kwargs of the function it wraps.

I think this mistake may already be made in other places.

Reply:
You might be right.
I probably should use 
```
s, node_or_state, args, kwargs = _get_callee_args(wrapped_args)
```
to detect the kwargs, just like permittedKwargs.

That's strange, because it worked fine for my testing.